Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilder.java

```diff
@@ -288,28 +287,28 @@ protected Conditions createConditions(final CallbackProperties properties) {
 
         // Not a *single* one of these methods ever returns a list with more than one element.
         // so in that case we can eventually drop the whole "List" thing going on and just add individual elements.
-        statements.addAll(createAuthenicationStatements(properties));
+        statements.add(createAuthenicationStatements(properties));
```

Comment:
This method can now return a null value here, we should have an appropriate null check before adding it to the list.